src/Teams/Controllers/MultipleAnswerQuestionController.cs

```diff
@@ -37,5 +37,59 @@ public IActionResult MultipleAnswerQuestionForm(string id, int[] answers)
             };
             return View(question);
         }
+
+        public IActionResult EditMultipleAnswerQuestion(Guid id)
+        {
+            var question = new MultipleAnswerQuestionViewModel()
+            {
+                SourceQuestion = questionRepository.PickById(id),
+                // ChosenOptions = answers
+            };
+            return View(question);
+        }
+
+        public IActionResult AddMultipleAnswerQuestion()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public IActionResult AddMultipleAnswerQuestion([FromBody] MAQDTOModel fromAjax)
+        {
+            List<MultipleAnswerQuestionOption> allAnswers = new List<MultipleAnswerQuestionOption>();
+            for (int i = 0; i < fromAjax.jTextList.Length; i++)
```

Comment:
Usually linq is easier to read. Could you try to replace the "for" loop with a linq and lambda expressions?